src/store/state.ts

```diff
@@ -47,7 +47,7 @@ const initialState: RootState = {
     notAskAgain: false,
   },
   experiments: {},
-  watsNewModalVersion: '',
+  whatsNewModalVersion: '',
```

Comment:
You have changed this but there is no migration, This means there will be an old state in the wallet called `watsNewModalVersion` 
Will this cause any bugs? I am happy without migration if it does not introduce errors when wallet updates.

Reply:
added inside the last migration now

Reply:
I dont see this variable being treated in the migration 😄  did you commit?